src/xrp-path-element.ts

```diff
@@ -0,0 +1,34 @@
+import { Payment } from './generated/web/org/xrpl/rpc/v1/transaction_pb'
+import XRPCurrency from './xrp-currency'
+
+type PathElement = Payment.PathElement
+
+/*
+ * A path in the XRP Ledger.
```

Comment:
```suggestion
 * A path element in the XRP Ledger.
```

Reply:
Also, is this a PathElement? Or a Path Step?
https://xrpl.org/paths.html#path-steps

Reply:
oh interesting that it's called a Path Step here in the documentation... It's called a PathElement in the rippled protocol buffers, so we'll stick with that for consistency.

Reply:
But I am going to update the comment/link to the path steps you have here